tests/v2/validation/rbac/clusterandprojectroles/README.md

```diff
@@ -0,0 +1,64 @@
+# Rbac
+
+## Getting Started
+Your GO suite should be set to `-run ^Test<>TestSuite$`. For example to run the cluster_role_test.go, set the GO suite to `-run ^TestClusterRoleTestSuite$` You can find specific tests by checking the test file you plan to run.
+Config needed for each of the suites cluster_role_test.go, project_role_test.go and restrictedadmin_role_test.go require the following config:
+
+```json
+"rancher": { 
+  "host": "rancher_server_address",
+  "adminToken": "rancher_admin_token",
+  "clusterName": "cluster_to_run_tests_on",
+  "insecure": true/optional,
+  "cleanup": false/optional,
+}
+```
+
+For the restrictedadmin_role_test.go run, we need the following additional paramters to be passed in the config file as we create a downstream cluster. We use rke1 custom cluster here: 
```

Comment:
is rke1 required? Is custom cluster required?

Reply:
for this test yes rke1 custom cluster config is required.